websites/ubuntu.lt/deploy.yml

```diff
@@ -97,19 +98,27 @@
     become_user: "{{ slug }}"
 
   - name: update translation files
-    command: "{{ home }}/venv/bin/django-admin compilemessages -l lt chdir={{ path }}/ubuntult"
+    shell:
+      cmd: "{{ home }}/venv/bin/django-admin compilemessages -l lt"
```

Comment:
NB: using `{{ shell_env_command_prefix }}` here fails because django-admin can't find `project.settings` on sys.path.

I suppose running manage.py would work due to python implicitly adding the manage.py's parent directory to sys.path, but ugh.